src/lib/serviceRoutines/statisticsTreat.cpp

```diff
@@ -240,13 +250,16 @@ std::string renderSemWaitStats(void)
   jh.addFloat("subCache",          semTimeCacheGet());
   jh.addFloat("connectionContext", mutexTimeCCGet());
   jh.addFloat("timeStat",          semTimeTimeStatGet());
+  jh.addFloat("metrics",           ((float) metricsMgr.semWaitTimeGet()) / 1000000);
```

Comment:
Interestingly, the alarms semaphore is not shown in the semWait output. I understand that we have a good reason for doing so and it is find, but I cannot remeber it... :)